plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -192,6 +192,8 @@ data DefaultFun
     -- Values
     | InsertCoin
     | UnionValue
+    | ValueData
+    | UnValueData
```

Comment:
I like these names more:
```suggestion
    | ValueAsData
    | DataAsValue
```

Reply:
The names are consistent with other Data builtins (IData, UnIData etc.)